app/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```

Comment:
This file should not be in the repo. Make sure you don't use `git add .` I always like to double check what files changed. 

You may need to add this file to `.gitignore`. This is a file that that lives in the project root.